tests/transactional/transactional_update.pm

```diff
@@ -36,11 +36,21 @@ sub get_utt_packages {
     assert_script_run "tar xzvf $tarball";
 }
 
-# Check that package presence & version is as expected
+=head2 check_package
+
+check_package([stage => $stage, package => $package]);
+
+Check that package presence & version is as expected
```

Comment:
s/presence/presences/

Reply:
was not my comment, I borrowed it, but the problem is in s/is/are/ due to presence is a noun in that sentence. fixed.